### DISCUSSION TITLE
Flux Webhook returning 400

### DISCUSSION THREAD

**Discussion by bojanraic** [onedr0p/cluster-template#1456] [Q&A] [answered]
My flux webhook keeps returning 400. I configured it on Github repo side, including `/hooks/<secret_value>`, same as was displayed in the Receiver status and it seemed fine. I used `application/json` as content type and tried with and without an explicit Secret value set to the same `<secret_value>` as above. In all those cases, a push event results in a 400 code and nothing is synced/reconciled as a result. 
I'm running out of ideas of what to double check or change to get it working so any suggestions are very welcome. 
Thanks!

**Comment by onedr0p** [comment]
Did you switch to the production acme servers as explained in the README?

**Reply by bojanraic** [reply]
Hi! I believe so, but I might have missed something...
Hope this helps 
```

certificate snippet: 
...
spec:
  issuerRef:
    name: letsencrypt-production
    kind: ClusterIssuer
  dnsNames:
  - example.com
  - '*.example.com'
  secretName: example-com-production-tls
  commonName: example.com
...

external ingress default certificate reference:
...
--default-ssl-certificate=network/example-com-production-tls
...

```

**Reply by onedr0p** [reply]
Are you able to access echo-server internally and externally?

**Reply by bojanraic** [reply]
Yup. I've accessed various external ingresses from other networks, including mobile networks/internet. Certificates seem to be working fine. 
I'll mess around with it some more tomorrow as it is late already.
I will gladly try any other suggestions and report back tomorrow. Thanks for helping!

**Reply by onedr0p** [reply]
Not sure what the issue could be, maybe cloudflare is blocking it? Do you have any firewall rules there?

**Reply by bojanraic** [reply, marked as answer]
Sigh... it was a PEBKAC situation...
I somehow managed to mess up the webhook token value in the webhook UI. Re-entered it correctly and voila!
Apologies for the inconvenience and thanks for your help!
